rom/dev/src/pcr.rs

```diff
@@ -39,26 +55,97 @@ pub fn extend_pcr0(env: &RomEnv) -> CaliptraResult<()> {
     // Lock the PCR from clear
     pcr_bank.map(|p| p.set_pcr_lock(caliptra_drivers::PcrId::PcrId0));
 
-    let extend = |data: Array4x12| {
+    let extend = |data: Array4x12, pcr_entry_id: PcrLogEntryId| {
```

Comment:
Optional but I feel like it might be useful to have the entry ID be measured as well, since it provides context for the bits that follow.

Reply:
We'll take this as a separate change. Can you file an issue for this?

Reply:
Filed at https://github.com/chipsalliance/caliptra-sw/issues/290